graphx/pom.xml

```diff
@@ -60,9 +60,8 @@
       <artifactId>guava</artifactId>
     </dependency>
     <dependency>
-      <groupId>com.github.fommil.netlib</groupId>
-      <artifactId>core</artifactId>
-      <version>${netlib.java.version}</version>
+      <groupId>dev.ludovic.netlib</groupId>
```

Comment:
Is your project GPL? that won't fly in an ASF project I'm afraid if so. (The relationship to the netlib code is already quite tricky w.r.t. licensing)

Reply:
It is currently GPLv2 with Classpath Exception, the same as the OpenJDK. I'm open to changing the licensing if it's a blocking issue.

Reply:
It would block including it, as the binary releases would include the software. If you're open to it, ALv2 is obviously the least contentious w.r.t. the ASF. Others are OK. Full story: https://www.apache.org/legal/resolved.html

Reply:
[v1.1.0](https://github.com/luhenry/netlib/tree/v1.1.0) relicenses to MIT.